src/styles/main.scss

```
@@ -0,0 +1,86 @@
+$calendar-box: 100px;
+$gap: 1px;
+$padding: 10px;
+$calendar-width: $calendar-box * 7 + $gap * 6 + $padding * 2;
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  margin: 0;
+  padding: $padding;
+  gap: $gap;
+  box-sizing: border-box;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $calendar-box;
+    width: $calendar-box;
+    list-style: none;
+    box-sizing: border-box;
+
+    background-color: #eee;
+    border: 1px solid #000;
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
+    }
+
+    @for $num from 1 through 31 {
+      &:nth-child(#{$num})::before {
+        content: '#{$num}';
+      }
+    }
+  }
+
+  $days: (
+    'mon': 0,
+    'tue': (
+      $calendar-box + $gap,
+    ),
+    'wed': (
+      $calendar-box * 2 + $gap * 2,
+    ),
+    'thu': (
+      $calendar-box * 3 + $gap * 3,
+    ),
+    'fri': (
+      $calendar-box * 4 + $gap * 4,
+    ),
+    'sat': (
+      $calendar-box * 5 + $gap * 5,
+    ),
+    'sun': (
+      $calendar-box * 6 + $gap * 6,
+    ),
```

The formula must be calculated inside the loop
```suggestion
    'tue': 1,
    'wed': 2,
    'thu': 3,
    'fri': 4,
    'sat': 5,
    'sun': 6,
```